### ISSUE TITLE
Loading nuxt.config.js file bug

### ISSUE THREAD

**Issue by the94air** [nuxt-community/hapi-nuxt#54]
Nuxtjs is ignoring my custom configuration of the `nuxt.config.js` in my project directory. Only in development mode but it is working just fine on production mode.
```js
await server.register({
  plugin: nuxtPlugin,
  options: {
    dev: true,
    rootDir: __dirname
  }
});
```

Here a simple reproduce of my issue: https://github.com/the94air/nuxt-hapi

**Comment by the94air** [comment]
I changed this line and everything seems to be working now. I couldn't get the tests to work on the fork so I can open PR. Sorry about that.
```diff
// Create nuxt instance using options
- const nuxt = new Nuxt(nuxtConfig)
+ const nuxt = new Nuxt(nuxtConfig.default)
```

**Comment by thefoxie** [comment]
Guess that's why my project stopped working after I changed module.exports to export default in nuxt.config.js. Your fix worked. Thank you @the94air

**Comment by thefoxie** [comment]
@pi0 could you look at it please?

**Comment by the94air** [comment]
I might close this issue soon because of no response :(

**Comment by thefoxie** [comment]
> I might close this issue soon because of no response :(

I wouldn't. I would love to get this merged because as of now I import this as git submodule :/

**Comment by the94air** [comment]
Same, I copied the file to my repo :b. But I am not sure if they can merge without working tests 🤔

**Comment by the94air** [comment]
hmmm

**Comment by pi0** [comment]
Hi there. Sorry for long delay. Issue should be fixed with [v3.0.0](https://github.com/nuxt-community/hapi-nuxt/releases/tag/v3.0.0)
